### PR TITLE
Add samples for python Flask and uWSGI

### PR DESCRIPTION
**Pull request by AevaOnline**
Neither sample works right now due to syscall limitations. This is documented within the sample. Uploading to share the test cases.

### CODE REVIEW

**Comment by jxyang** [comment]
Could you retry with #267  merged? @AevaOnline

**Comment by bodzhang** [comment]
With [#267](https://github.com/deislabs/mystikos/pull/267) merged, Flask sample test now fails due to Python 3 subprocess package [calling fork/exec](https://github.com/python/cpython/blob/76cd81d60310d65d01f9d7b48a8985d8ab89c8b4/Lib/subprocess.py#L1774). 

Python subprocess will [use POSIX_SPAWN on glibc version 2.24 and above ](https://github.com/python/cpython/blob/76cd81d60310d65d01f9d7b48a8985d8ab89c8b4/Lib/subprocess.py#L678). But currently Mytikos implementation doesn't support Ubuntu/glibc Python3 yet.

**Comment by bodzhang** [comment]
With [#267](https://github.com/deislabs/mystikos/pull/267) merged, Python uWSGI server sample works.  I did need to add "protocol = http" to uwsgi.ini file.

**Comment by bodzhang** [comment]
Setting Flask to not use multiple process, Flask server also works.  In flask_app.py:

`app.run(debug=True, host='0.0.0.0', port='8000', use_reloader=False)`

File: solutions/python_web_frameworks/Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ifdef STRACE
+OPTS = --strace
+endif
+
+all: app 
+
+app:
+	$(APPBUILDER) -f cpio -o hello_server.cpio Dockerfile 
+
+#run:
```
Comment (bodzhang) [inline solutions/python_web_frameworks/Makefile:15]: Replace with the following:

run:
	test -f server.pid && kill -9 `cat server.pid` || true
	@echo "===start flask server"
	@$(MYST_EXEC) $(OPTS) rootfs /usr/local/bin/python3 /app/flask_app.py & echo $$! > server.pid
	@sleep 10
	curl 127.0.0.1:8000 -w "\n"
	@echo "===stop flask server"
	@test -f server.pid && kill -9 `cat server.pid` && rm server.pid || true
	@echo "===start uwsgi server"
	@$(MYST_EXEC) $(OPTS) rootfs /usr/local/bin/uwsgi /app/uwsgi.ini & echo $$! > server.pid
	@sleep 10
	curl 127.0.0.1:8000 -w "\n"
	@test -f server.pid && kill -9 `cat server.pid` && rm server.pid || true
	@echo "===stop flask server"

server-flask:
	$(MYST_EXEC) $(OPTS) rootfs /usr/local/bin/python3 /app/flask_app.py

server-uwsgi:
	$(MYST_EXEC) $(OPTS) rootfs /usr/local/bin/uwsgi /app/uwsgi.ini

**Review by bodzhang — CHANGES_REQUESTED** [review]

**Comment by AevaOnline** [comment]
Thanks for the suggestions, @bodzhang ! Incorporated and tested and seems to be passing locally!

**Review by bodzhang — APPROVED** [review]

**Comment by bodzhang** [comment]
@AevaOnline , just realized that the PR did not add the new solution to /solutions/Makefile. Can you add the needed Makefile change?

**Review by bodzhang — APPROVED** [review]
Can address the needed  change in solutions directory Makefile in a separate PR.

**Pull request merged**